### PR TITLE
Queue: linked list implementation

### PR DESCRIPTION
**Pull request by antonioaversa**

### CODE REVIEW

File: MoreStructures/Stacks/IStack.cs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-﻿namespace MoreStructures.Stack;
+﻿namespace MoreStructures.Stacks;
```
Comment (andreaguarino) [inline MoreStructures/Stacks/IStack.cs:1]: why this change?

File: MoreStructures/Queues/LinkedListQueue.cs [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+/// <summary>
+/// A <see cref="IQueue{T}"/> implementation based on a singly-linked linked list of items.
+/// </summary>
+/// <typeparam name="T"><inheritdoc cref="IQueue{T}"/></typeparam>
+/// <remarks>
+///     <para id="advantages">
+///     ADVANTAGES AND DISADVANTAGES
+///     <br/>
+///     - The advantages an disadvantages of using a linked list over an array-based implementation are the same
```
Comment (andreaguarino) [inline MoreStructures/Queues/LinkedListQueue.cs:11]: an -> and

**Review by andreaguarino — APPROVED** [review]

File: MoreStructures/Stacks/IStack.cs (hunk 1 above)
Comment (antonioaversa) [reply MoreStructures/Stacks/IStack.cs:1]: To be coherent with the convention adopted on all other namespaces of the project.

In general, namespaces in C# are typically in plural form, especially when they contain multiple entities or implementations of one or more interfaces: e.g. `System.Collections`.

It's also a CONSIDER practice in Microsoft namespace guidelines (https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-namespaces).
